doc/conf.py

```diff
@@ -108,6 +108,50 @@
 
 napoleon_use_param = True
 napoleon_use_rtype = True
+napoleon_type_aliases = {
+    # general terms
+    "sequence": ":term:`sequence`",
+    "hashable": ":term:`hashable`",
+    "iterable": ":term:`iterable`",
+    "callable": ":py:func:`callable`",
+    "dict_like": ":term:`dict-like <mapping>`",
+    "dict-like": ":term:`dict-like <mapping>`",
+    "mapping": ":term:`mapping`",
+    "file-like": ":term:`file-like <file-like object>`",
+    # special terms
+    "nested list": "nested :class:`list`",
+    # stdlib type aliases
+    "MutableMapping": "~collections.abc.MutableMapping",
+    "sys.stdout": ":obj:`sys.stdout`",
+    "timedelta": "~datetime.timedelta",
+    # numpy terms
+    "array_like": ":term:`array_like`",
+    "array-like": ":term:`array-like <array_like>`",
+    # "scalar": ":term:`scalar`",
```

Comment:
does anyone know a linkable page that defines what we mean with `scalar`? I don't think `numpy`'s documentation does.